templates/homebrew.mxcl.buildkite-agent.plist.j2

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+
+<!-- https://launchd.info -->
+
+<plist version="1.0">
+<dict>
+  <key>EnvironmentVariables</key>
+  <dict>
+    <key>PATH</key>
```

Comment:
Ahh, my bad! Didn't read the docs through enough, this will set the `--spawn` flag for us and no need for the `--spawn={{ buildkite_agent_spawn }}` which I see you removed.